diff -urN libgnome-2.1.4.orig/schemas/desktop_gnome_interface.schemas libgnome-2.1.4/schemas/desktop_gnome_interface.schemas
--- libgnome-2.1.4.orig/schemas/desktop_gnome_interface.schemas	Mon Dec  9 23:58:39 2002
+++ libgnome-2.1.4/schemas/desktop_gnome_interface.schemas	Tue Dec 10 00:11:11 2002
@@ -29,7 +29,8 @@
     </schema>
     <schema>
       <key>/schemas/desktop/gnome/interface/enable_animations</key>
-      <applyto>/desktop/gnome/interface/enable_animations</apply>
+      <applyto>/desktop/gnome/interface/enable_animations</applyto>
+      <type>bool</type>
       <owner>gnome</owner>
       <default>true</default>
       <locale name="C">
